tests/kolainst/nondestructive/misc.sh

```diff
@@ -65,6 +68,10 @@ rpm-ostree status --jsonpath '$.deployments[0].booted' > jsonpath.txt
 assert_file_has_content_literal jsonpath.txt 'true'
 echo "ok jsonpath"
 
+rpmostree_busctl_call_os ListRepos > out.txt
+assert_file_has_content_literal out.txt '"is-devel" b false "is-enabled" b true "id" s "libtest" "description" s "libtest repo" "is-source" b false'
```

Comment:
Minor: is the order here well defined? Might be safer to grep for each kv individually instead.

Reply:
No idea about the order, I used an approach from https://github.com/coreos/rpm-ostree/commit/4c1218795e7e0571f22a5c86a3fc51f576565a18 but I agree with you, I'll split it to multiple calls